packages/block-directory/src/lock-unlock.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+
+export const { lock, unlock } =
+	__dangerousOptInToUnstableAPIsOnlyForCoreModules(
+		'I know using unstable features means my theme or plugin will inevitably break in the next version of WordPress.',
+		'@wordpress/edit-site'
```

Comment:
Shouldn't this be `@wordpress/block-directory` ?

Reply:
Yep, errant copy/pasta.